sqlite.go

```diff
@@ -130,6 +131,18 @@ func OpenConn(path string, flags OpenFlags) (*Conn, error) {
 	return openConn(path, flags)
 }
 
+func Crypto(conn *Conn, key string) error {
+	ckey := C.CString(key)
+	defer C.free(unsafe.Pointer(ckey))
+
+	// convert CString to void*: https://jamesadam.me/2016/03/26/c-and-go-dealing-with-void-parameters-in-cgo/
+	res := C.sqlite3_key(conn.conn, unsafe.Pointer(&ckey), 0)
+	if res != C.SQLITE_OK {
+		return fmt.Errorf("something went wrong")
```

Comment:
At minimum errors need to indicate what went wrong or where. In this package we have a function `reserr` to help format errors. This should be `return reserr("Conn.Unlock", "", "", res)`.

Reply:
Fixed the error issue